src/pages/admin/submissionData/[id].tsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import { api } from "../../../utils/api";
+import { useRouter } from "next/router";
+import DefaultLoadingPage from "~/components/loading/loading";
+import { Paper, Typography } from "@mui/material";
+
+const SubmissionId = () => {
+  const router = useRouter();
+  const { data, error, isLoading } =
+    api.collection.getSubmissionDataById.useQuery({
+      id: router.query.id as string,
+    });
+
+  if (isLoading) return <DefaultLoadingPage />;
+  if (error) return <div>Error: {error.message}</div>;
+
+  if (!data || !data.data) return <div>No data</div>;
+
+  console.log(data);
+
+  const {
+    firstName,
+    background,
+    lastName,
+    age,
+    dateOfBirth,
+    linkedInProfile,
+    email,
+    phoneNumber,
+    major,
+    minor,
+    ethnicityLevel1,
+    ethnicityLevel2,
+    school,
+    gender,
+    schoolState,
+    stateOfResidency,
+    schoolYear,
+    expectations,
+    hadMentor,
+  }: {
+    firstName: string;
+    background: Array<string>;
+    lastName: string;
+    age: number;
+    dateOfBirth: string;
+    linkedInProfile: string;
+    email: string;
+    phoneNumber: string;
+    major: string;
+    minor: string;
+    ethnicityLevel1: Array<string>;
+    ethnicityLevel2: string;
+    school: string;
+    gender: string;
+    schoolState: string;
+    stateOfResidency: string;
+    schoolYear: string;
+    expectations: string;
+    hadMentor: boolean;
+  } = data.data as {
```

Comment:
I guess the point of this component was to support any type of json, not just this one form - we can use this for now but in the future it should support any form with support for showing the results of any form.